src/propeller/core.cljc

```diff
@@ -3,18 +3,26 @@
   (:require [propeller.gp :as gp]
             [propeller.problems.simple-regression :as regression]
             [propeller.problems.string-classification :as string-classif]
-            [propeller.problems.software.number-io :as number-io]
-            [propeller.problems.software.smallest :as smallest]
             #?(:cljs [cljs.reader :refer [read-string]])))
 
+(defn eval-problem-var
+  [problem-name var-name]
+  (eval (symbol (str "propeller.problems." problem-name "/" var-name))))
+
 (defn -main
   "Runs propel-gp, giving it a map of arguments."
   [& args]
+  (when (empty? args)
+    (println "You must specify a problem to run.")
```

Comment:
Hopefully this is a helpful error message. Suggestions on how to make it more useful would certainly be appreciated. Do we want a full usage statement here, e.g., mentioning the ability to set parameters?